packages/teleport/src/DesktopSession/DesktopSession.tsx

```diff
@@ -102,6 +111,14 @@ export function DesktopSession(props: State) {
       based browsers like Google Chrome or Microsoft Edge. Brave users can \
       use the feature by navigating to brave://flags/#file-system-access-api \
       and selecting "Enable". Please switch to a supported browser.';
+    } else if (
+      fetchAttempt.status === 'processing' &&
+      tdpConnection.status === 'success'
+    ) {
+      errorText =
```

Comment:
The snapshot output for the `invalid` test doesn't appear to be correct as it doesn't have this content.

Reply:
Good catch: a47b801008c7ae5621425b79327a96bc73d7c55e